developer-docs-site/static/move_demo/client/index.js

```diff
@@ -0,0 +1,115 @@
+const aptos = require("aptos");
```

Comment:
could you use TS for this please, like some of the other demos?

Reply:
addressed the comments